### PR TITLE
Refactoring-and-Documentation

### PR DESCRIPTION
**Pull request by tomasztlalka**

### CODE REVIEW

File: POM/todo-angular-ts.ts [hunk 1]
```diff
@@ -79,13 +77,14 @@ export class AngularHomepage {
      * Add a specified number of todos with names of the format baseText followed by a sequence number between 1 and count.
      * @param count The number of todos to create.
      * @param baseText The text each todo should contain before its sequence number.
+     * @param enumerateTodos optional flag to disable todo string enumeration
```
Comment (Gandalfhfd) [inline POM/todo-angular-ts.ts:80]: Would benefit from stating explicitly whether true or false disables string enumeration

File: utils/helpers.ts [hunk 2]
```diff
@@ -4,13 +4,19 @@ export class MyHelpers {
      * Create an array of strings from some baseText, appended with its index + 1
```
Comment (Gandalfhfd) [inline utils/helpers.ts:4]: Now the function has been updated, the text might not be appended with this index

File: POM/todo-angular-ts.ts (hunk 1 above)
Comment (Gandalfhfd) [reply POM/todo-angular-ts.ts:80]: would be good explaining what the default behaviour is, too, actually

File: utils/helpers.ts [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
     /**
      * Create an array of strings from some baseText, appended with its index + 1
      * @param numberOfElements the number of elements in the string array
      * @param baseText the text from which to create the array
+     * @param enumerateStrings optional flag to disable array string enumeration
      * @returns Array(baseText+1, baseText+2, ..., baseText+numberOfElements)
      * @remarks uses type coalescing to allow us to add string and number
      */
-    async createArrayOfEnumeratedStrings(numberOfElements: number, baseText: string): Promise<string[]> {
+    async createArrayOfEnumeratedStrings(numberOfElements: number, baseText: string, enumerateStrings?: boolean): Promise<string[]> {
```
Comment (Gandalfhfd) [inline utils/helpers.ts:10]: consider changing the name of the function, now that it might not be enumerating them

**Review by Gandalfhfd — APPROVED** [review]

**Review by KnightSky4 — APPROVED** [review]

**Pull request merged**